make build: zip: not found

I checkout the current develop I start a Docker environment with `dev/start_dev_environment.sh` and then run `make build`. Then it fails with:

    Saved ./build/vendor/wheels/urllib3-1.25.11-py2.py3-none-any.whl
    Successfully downloaded backoff certifi chardet dataclasses httplib2 idna jinja2 markupsafe omegaconf psycopg2-binary pyparsing pyyaml requests typing-extensions urllib3
    git rev-parse --short HEAD > build/.commit
    # git archive -o source.tar HEAD
    git ls-files | tar Tcf - source.tar
    tar xf source.tar -C build/ --exclude=.commit
    rm source.tar
    cd build && rm -rf .github/ .gitignore .pylintrc .travis.yml* Makefile *.in tests/ dev/
    cd build && zip -r  -9 ../dist/cf-mendix-buildpack.zip .
    /bin/sh: 1: zip: not found
    make: *** [Makefile:30: build] Error 127
    root@5b6946d9958b:/app#

--- a/dev/Dockerfile
+++ b/dev/Dockerfile
@@ -11,6 +11,6 @@ RUN make install_requirements
 RUN wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | apt-key add - \
  && echo "deb https://packages.cloudfoundry.org/debian stable main" | tee /etc/apt/sources.list.d/cloudfoundry-cli.list \
  && apt-get update \
- && apt-get install -y cf-cli docker
+ && apt-get install -y cf-cli docker zip
 
 RUN cf install-plugin -f cflocal

